build/includes/sdk.mk

```diff
@@ -111,25 +111,25 @@ ensure-build-sdk-image:
 
 # Run SDK conformance Sidecar server in docker in order to run
 # SDK client test against it. Useful for test development
-SECONDS ?= 30
+run-sdk-conformance-local: TIMEOUT ?= 30
 run-sdk-conformance-local:
 	docker run -e "ADDRESS=" -p 59357:59357 \
-	 -e "TEST=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch" -e "TIMEOUT=$(SECONDS)" $(sidecar_tag)
+	 -e "TEST=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch" -e "TIMEOUT=$(TIMEOUT)" $(sidecar_tag)
 
 # Run SDK conformance test for a specific SDK_FOLDER
-SECONDS ?= 30
-run-sdk-conformance-test: SDK_FOLDER=$(SDK_FOLDER)
+run-sdk-conformance-test: TIMEOUT ?= 30
+run-sdk-conformance-test: TESTS ?= ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch
 run-sdk-conformance-test: ensure-build-sdk-image
 	sleep 2s && DOCKER_RUN_ARGS="--network=host $(DOCKER_RUN_ARGS)" COMMAND=sdktest $(MAKE) run-sdk-command & \
 	docker run -p 59357:59357 -e "ADDRESS=" \
-	 -e "TEST=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch" -e "TIMEOUT=$(SECONDS)" --net=host  $(sidecar_tag)
+	 -e "TEST=$(TESTS)" -e "TIMEOUT=$(TIMEOUT)" --net=host  $(sidecar_tag)
 
 # Run a conformance test for all SDKs supported
 run-sdk-conformance-tests: ensure-agones-sdk-image
 	$(MAKE) run-sdk-command SDK_FOLDER=node COMMAND=build
 	$(MAKE) run-sdk-command SDK_FOLDER=rust COMMAND=build
 	$(MAKE) run-sdk-conformance-test SDK_FOLDER=node
-	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=go TESTS=ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve
```

Comment:
Does the reserve test currently fail for the node & rust sdks? If so, are you going to fix that in a follow-up PR?

Reply:
Reserve is not implemented in the other SDKs (yet). I can take on the languages that I know how to write. 

On a broader scope, there is missing functionality from some of the SDKs. In the past, I've started with the Go SDK, and the community has written the updates to the other languages - but we might want to take a review, and fill in any gaps we find.

Reply:
So... to answer my original question, it sounds like the conformance test do fail in the other sdks (indicating that they haven't implemented Reserve), right?

Can you open up an issue to audit the existing SDKs for gaps and we can then prioritize getting them up to snuff prior to the 1.0 release.

Reply:
Yes, 100% correct.

I'll do a review, and write up a ticket right now :+1:

Reply:
Done - #927